workflow/controller/operator_concurrency_test.go

```diff
@@ -34,22 +35,22 @@ data:
 const wfWithSemaphore = `
 apiVersion: argoproj.io/v1alpha1
 kind: Workflow
-metadata: 
+metadata:
   name: hello-world
   namespace: default
-spec: 
+spec:
   entrypoint: whalesay
-  templates: 
-    - 
-      synchronization: 
-        semaphore: 
-          configMapKeyRef: 
+  templates:
+    -
+      synchronization:
```

Comment:
```suggestion
    - synchronization:
```

nit: this new line already existed, but is unnecessary.

or better would be to move `name: whalesay` up here actually